hack/kind-cluster-build.sh

```diff
@@ -215,7 +215,7 @@ kubectl apply -f ${ROOT}/manifests/local-volume-provisioner.yaml
 kubectl apply -f ${ROOT}/manifests/tiller-rbac.yaml
 
 kubectl create ns chaos-testing
-helm init --service-account=tiller --wait
+[[ $(helm version --short) != v3* ]] && helm init --service-account=tiller --wait
```

Comment:
I think it's better that we check whether helm version is less then v3.x?

Reply:
+1

Reply:
Or check helm version matches `v2*` ?  I'm not sure the command works for helm 1.x.

Reply:
Helm 1.x doesn't matter, because we require Helm version >= v2.8.2 at [Prerequisites](https://github.com/pingcap/chaos-mesh#prerequisites)